DashSync/shared/Models/Managers/Chain Managers/DSMasternodeManager.m

```diff
@@ -184,13 +184,15 @@ - (uint32_t)estimatedMasternodeListsToSync {
 }
 
 - (double)masternodeListAndQuorumsSyncProgress {
-    double amountLeft = self.masternodeListRetrievalQueueCount;
-    double maxAmount = self.masternodeListRetrievalQueueMaxAmount;
-    if (!amountLeft) {
-        return self.store.masternodeListsAndQuorumsIsSynced;
+    @synchronized (self) {
```

Comment:
@pankcuf I don't think `@synchronized` is a real solution, I personally against using `@synchronized` because it can lead performance decreasing and other "_unknowns_"